--- "a/CMakeLists.orig.txt"
+++ "b/CMakeLists.txt"
@@ -16,7 +16,8 @@ file (GLOB GENERAL RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
     [^.]*.cmake
     )
 
-include_directories(${PLUGIN_INCLUDE_DIRS})
+include_directories(${PLUGIN_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
+link_directories(${PYTHON_LIBRARIES})
 
 # Generated files are stored in ${GENERATED} by the project configuration
 SET_SOURCE_FILES_PROPERTIES(
@@ -36,3 +37,15 @@ SET( SOURCES
 
 # This will include Win/projectDef.cmake, Linux/projectDef.cmake, etc
 include_platform()
+
+target_link_libraries(${PLUGIN_NAME} fbxlib fbx_py)
+
+add_custom_command(
+    TARGET ${PLUGIN_NAME}
+    PRE_BUILD
+    COMMAND ${PATCH_EXECUTABLE} -t ${FIREBREATH_BINARY_DIR}/projects/${PLUGIN_NAME}/gen/FireBreathWin.def ${FBX_PY_ROOT}/patches/FireBreath.def.patch
+	COMMENT "Patching library export definition file"
+	VERBATIM
+)
+
+                
\ No newline at end of file
